app/src/main/java/com/example/justi/markthat/Home.java

```diff
@@ -10,4 +12,9 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_home);
     }
+
+    public void record(View view) {
```

Comment:
Is this going to record or recording? If its going to record then name it like goToRecord

Reply:
fixed